From f16832d3304a9bb3f103334103c9be29aec6699f Mon Sep 17 00:00:00 2001
From: De Huo <De.Huo@windriver.com>
Date: Thu, 25 Jul 2019 18:09:20 +0800
Subject: [PATCH] Remove accounts_password_pam_retry and add
 cracklib_accounts_password_pam_retry in WRLinux1019 profile

Replace rule accounts_password_pam_retry with rule
 cracklib_accounts_password_pam_retry in WRLinux1019 profile.

Upstream-Status: Pending

Signed-off-by: De Huo <De.Huo@windriver.com>
---
 wrlinux1019/profiles/draft_stig_wrlinux_disa.profile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/wrlinux1019/profiles/draft_stig_wrlinux_disa.profile b/wrlinux1019/profiles/draft_stig_wrlinux_disa.profile
index 9f461c6..c1c79a2 100644
--- a/wrlinux1019/profiles/draft_stig_wrlinux_disa.profile
+++ b/wrlinux1019/profiles/draft_stig_wrlinux_disa.profile
@@ -195,7 +195,7 @@ selections:
     - audit_rules_dac_modification_removexattr
     - audit_rules_dac_modification_fremovexattr
     - audit_rules_dac_modification_lremovexattr
-    - accounts_password_pam_retry
+    - cracklib_accounts_password_pam_retry
     - accounts_password_pam_ucredit
     - accounts_password_pam_lcredit
     - accounts_password_pam_dcredit
-- 
1.9.1

